### DISCUSSION TITLE
nuxt3 nuxi build throw Error

### DISCUSSION THREAD

**Discussion by Verdicts-ye** [vuejs/pinia#1587] [Help and Questions]
### Reproduction

no

### Steps to reproduce the bug

nuxt.config.ts file 
modules: [
    '@pinia/nuxt',
    '@vueuse/nuxt',
    '@unocss/nuxt',
    '@nuxtjs/color-mode',
  ],

yarn build 

### Expected behavior

build success

### Actual behavior

Rollup error: Could not load [projectpath]\node_modules\pinia\dist\pinia.mjs\dist\pinia.mjs (imported by .nuxt/dist/server/server.mjs): ENOENT: no such file or directory, open [projectpath]\node_modules\pinia\dist\pinia.mjs\dist\pinia.mjs'

**Comment by maicss** [comment]
see https://github.com/vuejs/pinia/discussions/1619
